src/tests/aeap.rs

```diff
@@ -345,7 +344,7 @@ async fn mark_as_verified(this: &TestContext, other: &TestContext) {
 }
 
 async fn get_last_info_msg(t: &TestContext, chat_id: ChatId) -> Option<Message> {
-    let msgs = chat::get_chat_msgs(&t.ctx, chat_id, constants::DC_GCM_INFO_ONLY)
+    let msgs = chat::get_chat_msgs_ex(&t.ctx, chat_id, true, false)
```

Comment:
Bool arguments, moreover if there are two or more, are not well-readable. A struct with named bool flags would be better

Reply:
works nicely only if you have an IDE that shows parameter names, indeed.